송하은/sugang-system/src/main/java/com/example/sugangsystem/controller/CourseController.java

```diff
@@ -0,0 +1,50 @@
+package com.example.sugangsystem.controller;
+
+import com.example.sugangsystem.dto.request.course.CourseSaveRequestDto;
+import com.example.sugangsystem.dto.request.course.CourseUpdateRequestDto;
+import com.example.sugangsystem.dto.response.course.CourseInfoResponseDto;
+import com.example.sugangsystem.dto.response.course.CourseListResponseDto;
+import com.example.sugangsystem.service.CourseService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/courses")
+public class CourseController {
+
+    private final CourseService courseService;
+
+    // 강의 생성
+    @PostMapping("/create")
```

Comment:
```suggestion
    @PostMapping
```
URI에는 동사를 지양하는 편입니다.